README.md

```diff
@@ -11,8 +11,9 @@
 ## Table of Contents
 
 1. [Requirements](#requirements)
-1. [Usage](#usage)
-1. [Development](#development)
+2. [Usage](#usage)
+3. [Development](#development)
+4. [Product-API] (#product-api)
```

Comment:
Just a heads up that, looking at the lack of highlighting, the 'Product-API' in the `[ ]` is not an active link to the 'product-api' section of your README